config/install/core.extension.yml

```diff
@@ -0,0 +1,54 @@
+_core:
```

Comment:
This field should not be in the module config/install directory. It's causing the module to not be able to install.

Reply:
I believe I have it in the correct place now. I found where the old `core.extension.yml` was located. It was just an empty file. So i replaced it with the new one. It should work now.

Reply:
The core.extension.yml file shouldn't be there at all in the module config/install directory. This will prevent the module from installing properly.

Reply:
Im not sure why it ended up in the modules config/install directory. But i moved it to where the empty file was located. In the web/core/config/install directory. Should it not be there?

Reply:
You don't need to move it anywhere since you copied it from the config sync directory in the first place. It should just be removed from this module's config/install directory and that removal should be committed to Git and pushed up to this repo. Then, I think you should be able to test that the module installs on a new Drupal site. If you try to install it on a new Drupal site now, an error appears in relation to this file.

Reply:
The file is deleted. It is ready to be tested again.